From stable-bounces@linux.kernel.org Wed Sep  6 11:10:14 2006
Message-Id: <20060906171922.357072000@localhost.localdomain>
Date: Wed, 06 Sep 2006 10:17:58 -0700
From: Stephen Hemminger <shemminger@osdl.org>
To: Jeff Garzik <jgarzik@pobox.com>
Content-Disposition: inline; filename=sky2-dotvers.patch
Cc: netdev@vger.kernel.org, stable@kernel.org
Subject: sky2: version 1.6.1

From: Stephen Hemminger <shemminger@osdl.org>

Since this code incorporates some of the fixes from 2.6.18, change
the version number.

Signed-off-by: Stephen Hemminger <shemminger@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/net/sky2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- linux-2.6.17.11.orig/drivers/net/sky2.c
+++ linux-2.6.17.11/drivers/net/sky2.c
@@ -51,7 +51,7 @@
 #include "sky2.h"
 
 #define DRV_NAME		"sky2"
-#define DRV_VERSION		"1.4"
+#define DRV_VERSION		"1.6.1"
 #define PFX			DRV_NAME " "
 
 /*
